Beginners/Lesson Five/Lesson Five.playground/Contents.swift

```diff
@@ -0,0 +1,240 @@
+/*:
```

Comment:
also include a workspace for digging around, could automate that too, only a simple XML file.

![screen shot 2016-09-10 at 8 32 47 pm](https://cloud.githubusercontent.com/assets/49038/18414434/be96c5a4-7795-11e6-8d5c-a95ab2121ce1.png)